crates/sui-benchmark/src/bin/stress.rs

```diff
@@ -144,7 +144,7 @@ async fn run(
 ) {
     let mut tasks = Vec::new();
     let (tx, mut rx) = tokio::sync::mpsc::channel(100);
-    let request_delay_micros = 1_000_000 / (opts.num_workers * opts.target_qps);
+    let request_delay_micros = (1_000_000 * opts.num_workers) / opts.target_qps;
```

Comment:
Hmm interesting.  So the more workers there are, the longer the expected delay....  because the cluster will be slower.  That makes sense

Reply:
correct, the more workers there are longer the delay per worker to achieve the targeted qps.